Resources/Prototypes/_LostParadise/Datasets/dreamverbs.yml

```
@@ -0,0 +1,1627 @@
+  - колебание
+  - удивляться
+  - труд
+  - переживать
+  - обёртывание
+  - крушение
+  - борьба
+  - изгиб
+  - зевок
+  - вопить
+  - молния
+  - увеличение
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)